src/components/app/components/styles.css

```diff
@@ -1,74 +1,77 @@
 .footer-container {
-    background-color: #f0f2f5;
-    text-align: center; 
-    color: green;  
+  background-color: #f0f2f5;
+  text-align: center;
+  color: green;
 }
 
 a {
-    color: green;
+  color: green;
 }
 
 .footer2 {
-    background-color: black;
-    text-align: center;
-    height: 50px;
-    padding-top: 15px;
-    font-size: 80%;
+  background-color: black;
+  text-align: center;
+  height: 50px;
+  padding-top: 15px;
+  font-size: 80%;
 }
 h4 {
-    color: azure;
-} 
+  color: azure;
+}
 
 .banner {
-    height: auto;
-    background-color: #106021; 
-    padding: 50px; 
-    text-align: center; 
-    color: white;
+  height: auto;
+  background-color: #106021;
+  padding: 50px;
+  text-align: center;
+  color: white;
 }
 .banner h2 {
-    color: white;
-    padding-bottom: 20px;
+  color: white;
+  padding-bottom: 20px;
 }
 
 .banner_flex {
-    display: flex; 
-    flex-wrap: wrap; 
-    justify-content: space-around;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
 }
 
 .banner_card {
-    width: 220px; 
-    display: block; 
-    text-align: center;
-    padding: 15px;
-    color: white;
+  width: 220px;
+  display: block;
+  text-align: center;
+  padding: 15px;
+  color: white;
 }
 
 .banner_icon {
-    border-radius:15px; 
-    width: 50px; 
-    height: 50px; 
-    background-color: white; 
-    margin: auto;
-
+  border-radius: 15px;
+  width: 50px;
+  height: 50px;
+  background-color: white;
+  margin: auto;
 }
 
 .banner_icon i {
-    color: #106021; 
-    font-size: 25px; 
-    padding-top: 15px;
-    font-weight: 7px;
+  color: #106021;
+  font-size: 25px;
+  padding-top: 15px;
+  font-weight: 7px;
 }
 
 .banner_color {
-    color: white;
+  color: white;
 }
 
 .banner button {
-    color: white;
-    background-color: #106021;
-    float: right;
-    margin-right: 100px;
-    margin-top: 40px;
-}
\ No newline at end of file
+  color: white;
+  background-color: #106021;
+  float: right;
+  margin-right: 100px;
+  margin-top: 40px;
+}
+
+.cart-count-nav > .bg-transparent {
+  background-color: transparent !important;
```

Comment:
it is not recommended to use `!important` , if there is a better way please remove it